[HousingAethernet] [HousingMapMarkerInfo] [HousingUnitedExterior]

IcarusTwine opened the pull request:
Added definitions for these and fixed a level link

IcarusTwine committed:
[HousingAethernet] [HousingMapMarkerInfo] [HousingUnitedExterior]

Added definitions for these and fixed a level link

ufx commented on SaintCoinach/ex.json:
```diff
@@ -7086,15 +7090,19 @@
         {
           "index": 5,
-          "name": "CustomTalk",
+          "name": "CustomTalk[1]",
```
You sure this one is right?  It doesn't appear to have any valid links back to CustomTalk.  Not sure about index 6.

IcarusTwine committed:
Corrected index 5/6

5 links to a sheet i dont think we have (similar to where Item.csv for aquarium links to nothing)
